lib/AST/Builtins.cpp

```diff
@@ -1625,6 +1625,20 @@ static ValueDecl *getAutoDiffAllocateSubcontext(ASTContext &ctx,
       ctx.TheRawPointerType);
 }
 
+static ValueDecl *getAutoDiffCreateLinearMapContext2(ASTContext &ctx,
```

Comment:
As the functions above with old builtins are never called from the compiler, I'd steal their names for the helpers that call new builtins

Reply:
I think it'll be better to keep the helpers symmetric with the names of the builtins. Otherwise `autoDiffProjectTopLevelSubcontext` will be the odd one out in different places.